example/ldk/index.ts

```diff
@@ -203,14 +209,12 @@ export const getTransactionData = async (
  */
 export const backupAccount = async (
 	account?: TAccount,
-): Promise<Result<string>> => {
+): Promise<Result<TAccountBackup>> => {
 	if (!account) {
 		account = await getAccount();
 	}
 	return await lm.backupAccount({
 		account,
-		setItem,
-		getItem,
```

Comment:
Yes so we are persisting everything natively now. We needed to change this to persist channel manager and channels natively because in the Swift/Kotlin the LDK guys pointed out that it's absolutely necessary to confirm that the state was persisted to disk in `update_persisted_channel(...)` or `persist_new_channel(...)` before LDK can continue processing events. Noticed this when LDK crashed right after the event to persist a channel was triggered but before it was actually saved with setItem. This caused the channel manager and the channel to be out of sync and so the node wasn't able to be restarted after the crash. The solution was to handle this synchronously on the native side. 

Then it felt a bit messy to be keeping half the state in files and half in async storage so opted to keep it all in one place.

So in your app you'll subscribe to the backup event and then once that is triggered you'll export state with `lm.backupAccount()` where you'll then do your own remote backup with icloud, gdrive or something else.

```javascript
ldk.onEvent(EEventTypes.backup, async () => {
  const backupResponse = await backupAccount();
  if (backupResponse.isErr()) {
	  return console.error(backupResponse.error);
  }
  
  //Save to cloud
  console.log(backupResponse.value);
 });
```

Only thing I might change soon is adding a debounce to the backup event. Noticed it gets called twice right after the first call because it's triggered on channel manager updates and channel updates. But will do this in a separate PR.